docs/source/user/index.rst

```diff
@@ -3,28 +3,76 @@
 User Documentation
 ==================
 
-This section contains documentation for the OpenFAST module-coupling environment and its underlying modules.
-Documentation covers usage of models, underlying theory, and in some cases module verification.
-
 We are in the process of transitioning legacy FAST v8 documentation, which can be found at https://nwtc.nrel.gov/.
-Details on the transition from FAST v8 to OpenFAST may be found in :numref:`fast_to_openfast`
 
+.. note::
+
+    Much of the documentation here is legacy documentation from FAST v8. While most of it is still
+    directly applicable to OpenFAST, portions may be out of date.
+
+
+.. _general-reference-docs:
+
+General
+~~~~~~~
 .. toctree::
    :maxdepth: 1
 
+   fast_to_openfast.rst
    api_change.rst
    input_file_overview.rst
-   aerodyn/index.rst
-   aerodyn-olaf/index.rst
-   aerodyn-aeroacoustics/index.rst
-   beamdyn/index.rst
-   subdyn/index.rst
-   elastodyn/index.rst
-   hydrodyn/index.rst
-   inflowwind/index.rst
-   servodyn/index.rst
-   servodyn-stc/StC_index.rst
-   fast_to_openfast.rst
-   cppapi/index.rst
-   fast.farm/index.rst
 
+Workshop material, legacy documentation, and other resources are listed below.
+
+- `Workshop Presentations <https://drive.google.com/drive/folders/1BDDfcnIyvmZCwf7eFo0ISI7aF_FMAOvt?usp=sharing>`_
+- :download:`FAST 8 User’s Guide <../../OtherSupporting/FAST8_UsersGuide.pdf>`
+- :download:`FAST v8 README <../../OtherSupporting/FAST8_README.pdf>`
+- `Implementation of Substructure Flexibility and Member-Level Load Capabilities for Floating Offshore Wind Turbines in OpenFAST <https://www.nrel.gov/docs/fy20osti/76822.pdf>`_
+- `FAST modularization framework for wind turbine simulation: full-system linearization <https://www.nrel.gov/docs/fy17osti/67015.pdf>`_
+- `Full-System Linearization for Floating Offshore Wind Turbines in OpenFAST <https://www.nrel.gov/docs/fy19osti/71865.pdf>`_ 
+- :download:`FAST with Labview <../../OtherSupporting/UsingFAST4Labview.pdf>`
+
```

Comment:
Can we add the old "FAST v8 Compiling Instructions" to this list?:
https://drive.google.com/file/d/1V2h6lA-wq5tqhiQR6uUDlLzVWltu5s1X/view?usp=sharing

Reply:
The first section on compiling for Windows with Intel in Visual Studio is relevant, but the rest is outdated. Instead of including this here and answering questions on the differences between this and our current build system, I prefer to include the relevant sections from this document in our existing section on compiling.